src/VisualStudio/Core/Test.Next/Services/ServiceHubServicesTests.cs

```diff
@@ -147,10 +147,13 @@ public async Task TestDesignerAttributes()
 
             using var connection = client.CreateConnection<IRemoteDesignerAttributeDiscoveryService>(callback);
 
+            // Actually pass in the document to scan.  Otherwise the test takes several seconds waiting to do the slow
+            // sweep of the entire solution.
```

Comment:
this was annoying me while running tests.

Reply:
Are we not using the expeditable delay thing for the underlying delay here?

Reply:
that would be another option.  it would require calling `public Task ExpeditedWaitAsync()`.  but i'm also ok doing that.  i'll update the code to that in a followup.